src/main/api/catalog.ts

```diff
@@ -85,17 +85,35 @@ export class CatalogApi implements ICatalogApi {
             }
         }
 
+        const isAudiobook = (item: PublicationView) =>
+            item.RDFType && /http[s]?:\/\/schema\.org\/Audiobook$/.test(item.RDFType);
+
+        const lastAdded = lastAddedPublicationViews.slice().filter((item) => !isAudiobook(item));
+        const lastRead = lastReadPublicationViews.slice().filter((item) => !isAudiobook(item));
+        const audiobooks = lastAddedPublicationViews.slice().filter(isAudiobook);
+        const lastAudiobooks = lastReadPublicationViews.slice().filter(isAudiobook);
```

Comment:
Is `.slice()` necessary, given that `.filter()` returns a separate array copy?

Reply:
no you're right